app/javascript/components/works/Filters.jsx

```diff
@@ -69,13 +69,16 @@ class Filters extends PureComponent {
                 <div className="checkbox-container">
                   {Object.keys(filters[type]).map(item => (
                     <div className="mb2 checkbox-item" key={item}>
-                      <input
-                        onClick={() => this.toggleCheckbox(type, item)}
-                        type="checkbox"
-                        className="checkbox"
-                        value={item}
-                      />
-                      <p className="capitalize">{convertSnakeCase(item)}</p>
+                      <label for={`checkbox-${item}`}>
+                        <input
+                          onClick={() => this.toggleCheckbox(type, item)}
+                          type="checkbox"
+                          className="checkbox"
+                          value={item}
+                          id={`checkbox-${item}`}
+                        />
+                        <p className="capitalize display-inline-block">{convertSnakeCase(item)}</p>
```

Comment:
Same thing here, we can use `ttc` from Tachyons instead of defining our own class heh (same thing as you used for dropdowns).

Go ahead and remove `capitalize` from `_fonts.scss`, too, since it looks like this is the only place it's used.